File: PASCAL/Cap07/c07ex07.pas
program c07ex07;

function IGUAL(A, B: Integer): Boolean;
begin
  IGUAL := A = B;
end;

var
  X, Y: Integer;

begin
  Write('Informe o 1o. valor: ');
  Readln(X);

  Write('Informe o 2o. valor: ');
  Readln(Y);

  if (IGUAL(X, Y)) then
    Writeln('Valores sao iguais')
  else
    Writeln('Valores sao diferentes');

  Writeln;
  Write('Tecle <Enter> para encerrar... ');
  Readln;
end.

File: GO/Cap07/c07ex07.go
// c07ex07.go

package main

import "fmt"

func Igual(A, B int) bool {
  return A == B
}

func main() {
  var X, Y int

  fmt.Print("Informe o 1o. valor: ")
  fmt.Scan(&X)

  fmt.Print("Informe o 2o. valor: ")
  fmt.Scan(&Y)

  if Igual(X, Y) {
    fmt.Println("Valores sao iguais")
  } else {
    fmt.Println("Valores sao diferentes")
  }

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
